nixos/modules/services/hardware/irexec.nix

```diff
@@ -0,0 +1,119 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.irexec;
+in {
+
+  ###### interface
+
+  options = {
+    services.irexec = {
+
+      enable = mkEnableOption "Handle events from IR remotes decoded by lircd(8)";
+
+      configs = mkOption {
```

Comment:
I think this option name should be reconsidered.

Modules often name their text-based option `extraConfig`, in contrast to `settings` which usually is freeform-based or at least nix-based configuration, and `configFile`.

Since there are no freeform generator for this format, I think `extraConfig` might be the way to go here, even though it makes less semantic sense.

Alternatively, just since this file seems like it will grow quite long, it might be worth just making it `configFile` and rather let people use `pkgs.writeText` if they want to inline it.